docs/src/pages/components/pickers/MaterialUIPickers.js

```diff
@@ -18,21 +19,27 @@ export default function MaterialUIPickers() {
     <LocalizationProvider dateAdapter={AdapterDateFns}>
       <Stack spacing={3}>
         <DesktopDatePicker
-          label="Date picker desktop"
+          label="Date desktop"
           inputFormat="MM/dd/yyyy"
           value={value}
           onChange={handleChange}
           renderInput={(params) => <TextField {...params} />}
         />
         <MobileDatePicker
-          label="Date picker mobile"
```

Comment:
component type in label is an anti-pattern. Screen reader will duplicate that information and sighted users also "see" what this component is.